membrane_curvature/base.py

```diff
@@ -68,8 +68,20 @@ class MembraneCurvature(AnalysisBase):
         Each array has shape (`n_x_bins`, `n_y_bins`)
 
 
+    See also
+    --------
+    `MDAnalysis.transformations.wrap 
+    <https://docs.mdanalysis.org/1.0.0/documentation_pages/transformations/wrap.html>`_
+
     Notes
     -----
+    Use `wrap=True` when `mda.Universe` contains the raw trajectory, and
+    `wrap=False` for processed trajectories. For more details on when to use
+    `wrap=True`, check the `Usage
+    <https://membrane-curvature.readthedocs.io/en/latest/source/pages/Usage.html>`_
+    page.
```

Comment:
Can this be a little more fine-grained -- do many engines unwrap atoms in raw trajectories? If the trajectory is raw, it's usually more likely that the atoms are already in the unit cell, right? Could you mention translation vs rotation?

Reply:
I changed the lines a bit. Happy to received suggestions for the pushed changes.